specs/lifespan.rst

```diff
@@ -122,14 +122,14 @@ Keys:
 
 
 Version History
-===============
+''''''''''''''''''''''''''''''''
```

Comment:
You need to make the length of these match the titles above them :) Once that's done, I'll happily merge it though - good catch.

Reply:
Okay, I finished the revision.